mod/update_contact.php

```diff
@@ -28,11 +28,10 @@
 
 function update_contact_content(App $a)
 {
-	if (!empty($_GET['force']) || !DI::pConfig()->get(local_user(), 'system', 'no_auto_update')) {
-		$text = Contact::content([], true);
+	if (!empty($a->argv[1]) && (!empty($_GET['force']) || !DI::pConfig()->get(local_user(), 'system', 'no_auto_update'))) {
+		$text = Contact::getConversationsHMTL($a, $a->argv[1], true, ($_GET['item'] ?? 0));
 	} else {
 		$text = '';
 	}
-
```

Comment:
Why did you remove this new line?

Reply:
Accident?

Reply:
Because not doing a thing is easier than doing it? I've been consistently adding a new line after closing curly braces so I wanted to know why you removed it in this specific instance.